src/values/FunctionValue.js

```diff
@@ -19,6 +19,11 @@ import invariant from "../invariant.js";
 export default class FunctionValue extends ObjectValue {
   constructor(realm: Realm, intrinsicName?: string) {
     super(realm, realm.intrinsics.FunctionPrototype, intrinsicName);
+    let currentReconciler = realm.react.currentReconciler;
+
+    if (realm.react.enabled && currentReconciler !== null && currentReconciler.evaluatedFunctions !== null) {
+      currentReconciler.evaluatedFunctions.add(this);
```

Comment:
Instead of pushing around this additional piece of state (evaluatedFunctions), why is it not possible to leverage the `createdObjects` in the effects, filtering for `FunctionValue`s?

Reply:
Good point, I'll try that tomorrow.